docs/database/sql-server.md

```diff
@@ -152,6 +153,21 @@ and SHOULD be provided **at span creation time** (if provided at all):
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+### Propagating context to SQL Server
+
+Instrumentations SHOULD propagate the context information to databases.
+
+| Attribute      | Type   | Description                           | Require level | Stability                                                      |
+|----------------|--------|---------------------------------------|---------------|----------------------------------------------------------------|
+| `service.name` | string | Logical name of the service [1]       | `Required`    | ![Development](https://img.shields.io/badge/-development-blue) |
+| `traceparent`  | string | The trace context of current span [2] | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+
+**[1] `service.name`:** MUST be the same for all instances of horizontally scaled services. If the value was not specified, SDKs MUST fall back to `unknown_service:` concatenated with [process.executable.name](https://opentelemetry.io/docs/specs/semconv/attributes-registry/process/), e.g. `unknown_service:bash`. If `process.executable.name` is not available, the value MUST be set to `unknown_service`.
+
+**[2] `traceparent`:** MUST be in the [text format](https://www.w3.org/TR/trace-context/#traceparent-header).
```

Comment:
As described before. I think this can be removed.

Reply:
updated